From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Subject: [PATCH v2] comdlg32: Improve PRINT32 dialog
Message-Id: <PS2P216MB0066F319A54FF3669F82BBA993970@PS2P216MB0066.KORP216.PROD.OUTLOOK.COM>
Date: Tue, 29 Jan 2019 01:06:06 +0000

- Corrected TAB order to be the same as windows
- Moved Help button to Bottom Right, align top with OK/Cancel buttons.

Signed-off-by: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
---
 dlls/comdlg32/comdlg32.rc | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/dlls/comdlg32/comdlg32.rc b/dlls/comdlg32/comdlg32.rc
index 22a5a93..e4a5a63 100644
--- a/dlls/comdlg32/comdlg32.rc
+++ b/dlls/comdlg32/comdlg32.rc
@@ -354,16 +354,12 @@ STYLE DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU |
       DS_CONTEXTHELP | DS_3DLOOK
 CAPTION "Print"
 FONT 8, "MS Shell Dlg"
-{
-    DEFPUSHBUTTON   "OK",     IDOK,     176,164, 50,14, WS_GROUP | BS_DEFPUSHBUTTON
-    PUSHBUTTON      "Cancel", IDCANCEL, 230,164, 50,14, WS_GROUP
-    PUSHBUTTON      "&Help",  pshHelp,  50, 161, 50,14, WS_GROUP
-
+BEGIN
     GROUPBOX        "Printer",        grp4,   8,  4, 272,84, WS_GROUP
-    CONTROL         "Print to fi&le", chx1, "Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,185,36,90,16
+    COMBOBOX                          cmb4,  52, 18, 152,152,CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_GROUP | WS_TABSTOP
     PUSHBUTTON      "&Properties",    psh2, 212, 17,  60,14, WS_GROUP
+    CONTROL         "Print to fi&le", chx1, "Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,185,36,90,16
     LTEXT           "&Name:",         stc6,  16, 20,  36,8
-    COMBOBOX                          cmb4,  52, 18, 152,152,CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_GROUP | WS_TABSTOP
     LTEXT           "Status:",        stc8,  16, 36,  47,10, SS_NOPREFIX
     LTEXT           "",               stc12, 65, 36, 120,10, SS_NOPREFIX | SS_LEFTNOWORDWRAP
     LTEXT           "Type:",          stc7,  16, 48,  47,10, SS_NOPREFIX
@@ -373,12 +369,6 @@ FONT 8, "MS Shell Dlg"
     LTEXT           "Comment:",       stc9,  16, 72,  47,10, SS_NOPREFIX
     LTEXT           "",               stc13, 65, 72, 212,10, SS_NOPREFIX | SS_LEFTNOWORDWRAP
 
-    GROUPBOX        "Copies",         grp2, 160, 92, 120,64, WS_GROUP
-    LTEXT           "Number of &copies:",stc5,168,105,68,8
-    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
-    CONTROL         "C&ollate",       chx2,"Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,168,118,100,12
-    EDITTEXT                          edt3, 240,103,  32,12, WS_GROUP | ES_NUMBER
-
     GROUPBOX        "Print range",    grp1,   8,92,  144,64, WS_GROUP
     CONTROL         "&All",           rad1,"Button",BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP,16,106,64,12
     CONTROL         "Pa&ges",         rad3,"Button",BS_AUTORADIOBUTTON,16,122,36,12
@@ -387,7 +377,17 @@ FONT 8, "MS Shell Dlg"
     EDITTEXT                          edt2, 120,122,  26,12, WS_GROUP | ES_NUMBER
     RTEXT           "&from:",         stc2,  52,124,  20,8
     RTEXT           "&to:",           stc3,  99,124,  20,8
-}
+
+    GROUPBOX        "Copies",         grp2, 160, 92, 120,64, WS_GROUP
+    LTEXT           "Number of &copies:",stc5,168,105,68,8
+    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
+    CONTROL         "C&ollate",       chx2,"Button",BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP,168,118,100,12
+    EDITTEXT                          edt3, 240,103,  32,12, WS_GROUP | ES_NUMBER
+
+    DEFPUSHBUTTON   "OK",     IDOK,     176,164, 50,14, WS_GROUP | BS_DEFPUSHBUTTON
+    PUSHBUTTON      "Cancel", IDCANCEL, 230,164, 50,14, WS_GROUP
+    PUSHBUTTON      "&Help",  pshHelp,  50, 164, 50,14, WS_GROUP
+END
 
 PRINT32_SETUP DIALOG  32, 32, 288, 178
 STYLE DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CONTEXTHELP | DS_3DLOOK

-- 
1.9.1

